namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-managing-prisoner-apps-prod/resources/hmpps-managing-prisoner-apps-api.tf

```diff
@@ -0,0 +1,16 @@
+module "hmpps_managing_prisoner_apps_api" {
+  source      = "github.com/ministryofjustice/cloud-platform-terraform-hmpps-template?ref=1.0.0"
+  github_repo = "hmpps-managing-prisoner-apps-api"
+  application = "hmpps-managing-prisoner-apps-api"
+  github_team = "hmpps-launchpad"
+  environment = var.environment # Should match environment name used in helm values file e.g. values-dev.yaml
+  reviewer_teams                = ["hmpps-launchpad"] # Optional team that should review deployments to this environment.
+  selected_branch_patterns      = ["main", "**/**", "**"] # Optional
```

Comment:
I'd suggest either commening this out and uncommenting line 9. Or change this to be just `["main"]`. We don't want any branches other than `main` going to prod.